modules/install-upgrade/pages/production-cluster-multiregion.adoc

```diff
@@ -0,0 +1,134 @@
+:activeTopics: 1
+:messageSize: 1000
+:messageThroughput: 100000
+:retentionPolicy: 3600
+:ttlPolicy: 24
+:tieredStoragePolicy: n/a
+:messageReplicationFactor: 3
+:clusterReplicationFactor: 2
+
+= Multi-region georeplication
+
+This page contains sizing and configuration information for a georeplicated VM-based production cluster. +
+Pulsar clusters can be configured to replicate data across multiple regions to maximize availability.
+
+We apply our sizing methodology with the following assumptions: +
+
+* Replication will be *asynchronous* and *will not* use a global Zookeeper.
```

Comment:
What does it mean by "asyn replication"? Do we explain it somewhere? If so, can we provided a link?